packages/angular_devkit/schematics/src/formats/app-name.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { schema } from '@angular-devkit/core';
+import { htmlSelectorRe } from './html-selector';
+
+
+const unsupportedProjectNames = ['test', 'ember', 'ember-cli', 'vendor', 'app'];
```

Comment:
Should the ember references be removed?  The existing CLI logic has them but that was really more of a carryover from the original ember code.

Reply:
Done